dist/index.html

```diff
@@ -1 +1,28 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>To do</title><link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"><script defer="defer" src="bundle.js"></script></head><body><div class="container"><p class="todo-header">Today's To Do</p><div class="todos"><input class="new-todo-input" placeholder="Add to your list..." style="font-style:italic"> <span class="material-icons btn-icon">keyboard_return</span></div><ul class="todo-list"></ul><div class="btn-item item"><button class="btn-clear-todos" type="button">Clear all completed</button></div></div></body></html>
\ No newline at end of file
+<!DOCTYPE html>
```

Comment:
- Remember that your files inside your `` dist `` directory, it's created by webpack after build the project in console.
- Kindly work from your `` index.html ``  file inside your  `` src `` directory.